platform/iphone/os_iphone.cpp

```diff
@@ -109,16 +107,22 @@ void OSIPhone::initialize(const VideoMode& p_desired,int p_video_driver,int p_au
 	supported_orientations |= ((GLOBAL_DEF("video_mode/allow_vertical", false)?1:0) << PortraitDown);
 	supported_orientations |= ((GLOBAL_DEF("video_mode/allow_vertical_flipped", false)?1:0) << PortraitUp);
 
-	rasterizer_gles22 = memnew( RasterizerGLES2(false, false, false) );
-	rasterizer = rasterizer_gles22;
-	rasterizer_gles22->set_base_framebuffer(gl_view_base_fb);
+//	rasterizer_gles22 = memnew( RasterizerGLES2(false, false, false) );
+//	rasterizer = rasterizer_gles22;
+//	rasterizer_gles22->set_base_framebuffer(gl_view_base_fb);
 
-	visual_server = memnew( VisualServerRaster(rasterizer) );
-	if (get_render_thread_mode() != RENDER_THREAD_UNSAFE) {
+//	visual_server = memnew( VisualServerRaster(rasterizer) );
 
-		visual_server = memnew(VisualServerWrapMT(visual_server, false));
-	};
-	visual_server->init();
+	RasterizerGLES3::register_config();
+	RasterizerGLES3::make_current();
+	RasterizerGLES3::set_system_fbo(gl_view_base_fb);
+
+	visual_server = memnew( VisualServerRaster() );
+	// FIXME: Reimplement threaded rendering? Or remove?
+//	if (get_render_thread_mode() != RENDER_THREAD_UNSAFE) {
```

Comment:
In case you're up for yet an `--amend`, it would be good to change the comment style to `/*` `...` `*/` around the block that should be disabled, as clang-format will mess things up here.